src/Main.java

```diff
@@ -1,114 +1,64 @@
-        for (char character : chars) {
-            if (letter == character) {
-                occurance += 1;
+        for (String word : words) {
+            if (word.equals(target)) {
+                occurance++;
             }
         }
-        System.out.println("Occurrence \"" + letter + "\" in " + str + " = " + occurance);
-
-    }
-
-    public static int findWordPosition(String source, String target) {
-        int pos;
-
-        pos = source.toLowerCase().indexOf(target.toLowerCase());
-        return pos;
+        return occurance;
     }
 
-    public static String stringReverse(String str) {
-        StringBuilder reversed = new StringBuilder(str);
-        reversed.reverse();
-        return reversed.toString();
+    public static List<Object> toList(Object[] toConvert) {
+        return new ArrayList<Object>(Arrays.asList(toConvert));
     }
 
-    public static boolean isPalindrome(String str) {
-        String reversed = stringReverse(str);
-        return reversed.equals(str);
+    public static List<Object> findUnique(Integer[] integers) {
+        Set<Object> unique = new HashSet<>(toList(integers));
+        return new ArrayList<>(unique);
     }
 
-    public static void game() {
-        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado", "broccoli", "carrot", "cherry", "garlic", "grape", "melon", "leak", "kiwi", "mango", "mushroom", "nut", "olive", " pea", "peanut", "pear", "pepper", "pineapple", "pumpkin", "potato"};
-        Random random = new Random();
-        int rundomNum = random.nextInt(words.length);
-
-//        System.out.println(rundomNum);
-
-        Scanner scanner = new Scanner(System.in);
-        System.out.println("Let`s play a game, try to guess the word I thought");
-        System.out.println("It`s one of these:");
-        for (String fruit : words) {
-            System.out.print(fruit + ", ");
-        }
-        boolean isGuessed = false;
-
-        char[] secretArr = new char[15];
-        Arrays.fill(secretArr, '#');
-
-        while (!isGuessed) {
+    public static void calcOccurance(List<String> words) {
+        Set<String> unique = new HashSet<>(words);
 
-            String secret = words[rundomNum];
-//            System.out.println(secret);
-            System.out.println("\nNow type your guess");
-            String guess = scanner.next();
-            guess = guess.trim();
-
-
-            if (guess.equals(secret)) {
-                System.out.println("Congrats, you won this game");
-                isGuessed = true;
-            } else {
-                helper(guess, secret, secretArr);
+        for (String word : unique) {
+            int currentOccurance = 0;
+            for (String word2 : words) {
+                if (word.equals(word2)) {
+                    currentOccurance++;
+                }
             }
+            System.out.println("\"" + word + "\"" + " occurance: " + currentOccurance);
```

Comment:
please, think how to use only one for statement, it will dramatically decrease the complexity of the code